### ISSUE TITLE
Feature: Letter highlighting (as with the original 8pen)

### ISSUE THREAD

**Issue by Hate9** [8VIM/8VIM#271]
I recently saw a video of the original 8pen keyboard, and something that struck me was that, when the keyboard was being used, the currently "active" letter (which letter would be typed if the user moved their finger to the center again) was highlighted by being a different color.
This seems like a great feature for people just learning the keyboard, and might cut down on learning-time.

**Comment by flide** [comment]
Yea... about that, I am really bad with android ui, I don't know how to do
that.

**Comment by Hate9** [comment]
I mean, the letters already have a configurable color, so I imagine it couldn't be that hard to change the color of one of the letters.
To be fair, I could be wrong, but that's how it seems to me.

**Comment by sslater11** [comment]
I made a pull request with MaethorNaur's letter highlighting. It looks a bit quick/confusing in the gif, but if you're the one typing, it really does help.

I have added a background behind the letter to make it stand out more. I think it looks good.

The background is the same color as the trail color, but it's a little lighter as the darker trail colors made it hard to see the black text.

![8vim letter background highlighting](https://user-images.githubusercontent.com/43177940/183313000-ce43a6b8-102e-4d7b-886d-accd57cc7a00.gif)
